Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.yml

```diff
@@ -0,0 +1,1901 @@
+        - contextPath: NSM.Alerts.Attacker.networkObject
+          description: The attacker network object.
+          type: Unknown
+        - contextPath: NSM.Alerts.Target.ipAddrs
+          description: The target IP address.
+          type: String
+        - contextPath: NSM.Alerts.Target.port
+          description: The target port.
+          type: Number
+        - contextPath: NSM.Alerts.Target.hostName
+          description: The target host name.
+          type: String
+        - contextPath: NSM.Alerts.Target.country
+          description: The target country.
+          type: String
+        - contextPath: NSM.Alerts.Target.os
+          description: The target operating system.
+          type: Unknown
+        - contextPath: NSM.Alerts.Target.vmName
+          description: The target VM name.
+          type: Unknown
+        - contextPath: NSM.Alerts.Target.proxyIP
+          description: The target proxy IP.
+          type: String
+        - contextPath: NSM.Alerts.Target.user
+          description: The target user.
+          type: Unknown
+        - contextPath: NSM.Alerts.Target.risk
+          description: The target risk.
+          type: String
+        - contextPath: NSM.Alerts.Target.networkObject
+          description: The target network object.
+          type: Unknown
+        - contextPath: NSM.Alerts.MalwareFile.fileName
+          description: The name of the malware file.
+          type: String
+        - contextPath: NSM.Alerts.MalwareFile.fileHash
+          description: The file hash of the malware file.
+          type: String
+        - contextPath: NSM.Alerts.MalwareFile.fileSHA1Hash
+          description: The malware file SHA1 hash.
+          type: String
+        - contextPath: NSM.Alerts.MalwareFile.fileSHA256Hash
+          description: The file SHA256 hash of the malware file.
+          type: Unknown
+        - contextPath: NSM.Alerts.MalwareFile.malwareName
+          description: The name of the malware.
+          type: String
+        - contextPath: NSM.Alerts.MalwareFile.malwareConfidence
+          description: Malware confidence
+          type: String
+        - contextPath: NSM.Alerts.MalwareFile.engine
+          description: Malware file engine.
+          type: String
+        - contextPath: NSM.Alerts.MalwareFile.engineId
+          description: Malware file engine ID.
+          type: Number
+        - contextPath: NSM.Alerts.MalwareFile.size
+          description: The malware file size.
+          type: Unknown
+        - contextPath: NSM.Alerts.MalwareFile.description
+          description: Malware file description.
+          type: Unknown
+        - contextPath: NSM.Alerts.MalwareFile.additionalReference
+          description: Malware file additional reference.
+          type: Unknown
+        - contextPath: NSM.Alerts.MalwareFile.cveId
+          description: Malware file CVE ID.
+          type: Unknown
+        - contextPath: NSM.Alerts.endpointExcutable.name
```

Comment:
In this context path, NSM.Alerts.endpointExcutable.name, Executable is misspelled.  Is that on purpose?  If not, it needs to be fixed in several places in this file.

Reply:
yes, it is written this way in the API 
https://docs.trellix.com/bundle/network-security-platform-9.1.x-manager-api-reference-guide/page/GUID-E9CC6871-867D-47DD-BC61-4ACD153BB412.html